javascript/javascript3/week3/shoppingCart.js

```diff
@@ -0,0 +1,46 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    this.products.push(product);
+  }
+
+  removeProduct(product) {
+    this.products.splice(this.products.indexOf(product));
+  }
+
+  searchProduct(productName) {
+    return this.products.filter((product) => product.name == productName);
```

Comment:
also cover the lower/upper case scenario. Searching will not work if the casing is different